keyboards/southpole/keymaps/foobeard/keymap.c

```diff
@@ -0,0 +1,78 @@
+/*
+Base Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Modified 2017 Andrew Novak <ndrw.nvk@gmail.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "southpole.h"
+
+enum layers {
+  _QWERTY,
+  _RAISE
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+#define ______ KC_TRNS
+#define RAISE MO(_RAISE)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	/* Layer 0, default layer
+	*  ,--------------------------------------------------------------------------------------------------------------------------------------------.
+	*  |NumLck|   /  |   *  |   -  | | Esc |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |    0 |   -  |   =  |   \  |Delete| Ins  |
+	*  |------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------|
+	*  |  7   |   8  |   9  |   +  | | Tab   |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |    P |   {  |   }  |   BkSpc   | Vol+ |
+	*  |------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------|
+	*  |  4   |   5  |   6  |   +  | | Caps   |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |    ; |   '  |      Return     | Vol- |
+	*  |------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------|
+	*  |  1   |   2  |   3  |  En  | | LShft     |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |    RShft   |   Up   | Mute |
+	*  |------+------+------+------+-+----------------------------------------------------------------------------------------------+--------+------|
+	*  |      0      |   .  |  En  | | Ctrl  |  Win  |  Alt  |                  Space                            |  Fn  |    | Left |  Down  | Right|
+	*  `------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------'
+	*/
+    [_QWERTY] = KEYMAP(
```

Comment:
Please change this to:

```suggestion
    [_QWERTY] = LAYOUT(
```